package.json

```
@@ -9,8 +9,12 @@
     "start": "cross-env NODE_ENV=development electron .",
     "lint": "eslint . && tsc -p tsconfig.json && tsc -p renderer/tsconfig.json && prettier --check .",
     "lint:fix": "eslint --fix . && prettier --write .",
-    "test": "run-s build:* && run-s test:*",
+    "test": "run-s build:* && run-s test:* && vitest run",
     "test:e2e": "xvfb-maybe cross-env NODE_ENV=test playwright test 'test/e2e/.*.test.js'",
+    "vitestpreview": "vite preview",
+    "vitest": "vitest run",
```

```suggestion
    "test": "run-s build:* && run-s test:*",
    "test:e2e": "xvfb-maybe cross-env NODE_ENV=test playwright test 'test/e2e/.*.test.js'",
    "test:ui": "npm run vitest",
    "vitestpreview": "vite preview",
    "vitest": "vitest run",
```

to mirror the existing structure more